packages/salesforce/src/Adaptor.js

```diff
@@ -510,7 +524,7 @@ export function destroy(sObject, attrs, options) {
 }
 
 /**
- * Create a new object.
+ * Create a new sObject record(s).
```

Comment:
Wait, this can do multiple records?

Is this confirmed behaviour? I see the `jsforce` supports  an array. I guess we just pass whatever through?

Basically if you're gonna drop a bombshell like `(s)` it needs documenting properly somewhere.

Reply:
I am glad you mentioned that, it turns out you can create a single record or multiple records using `create, update or upsert` function. So i have update the documentation accordingly. Please have another look